convert2rhel/main.py

```diff
@@ -227,10 +240,10 @@ def post_ponr_conversion():
 
     loggerinst.task("Convert: Import Red Hat GPG keys")
     pkghandler.install_gpg_keys()
+    loggerinst.task("Convert: Replace system packages")
+    pkghandler.transaction_handler.process_transaction()
     loggerinst.task("Convert: Prepare kernel")
     pkghandler.preserve_only_rhel_kernel()
```

Comment:
Check with @bocekm or @SpyTec if the call for `process_transaction()` can be before of the `preserve_only_rhel_kernel()` or not.